src/main/java/org/cloudburstmc/server/Server.java

```diff
@@ -1705,7 +1707,7 @@ private void registerVanillaComponents() {
         this.defaultLevelData.getGameRules().putAll(this.gameRuleRegistry.getDefaultRules());
     }
 
-    private void loadLevels() throws IOException {
+    private void loadWorlds() throws IOException {
```

Comment:
this method is loading `Level` objects, not worlds.

Reply:
Got it, will fix both issues

Reply:
If I may ask though, is there a difference between Level objects and Worlds?